### DISCUSSION TITLE
can't import whisper to python

### DISCUSSION THREAD

**Discussion by zenku-desu** [openai/whisper#1272] [Q&A] [answered]
I AM ON WINDOWS 10

I am trying to add the whisper to my 3.10.10 python script and when I try to import it it does not find it saying 

Import "whisper" could not be resolved
it is in the image shown below

![image](https://user-images.githubusercontent.com/131547251/233797231-e2804fc9-f7f5-4819-9410-b47200a86b7a.png)

I installed it using 

pip install git+https://github.com/openai/whisper.git 

and then update it using 

pip install --upgrade --no-deps --force-reinstall git+https://github.com/openai/whisper.git

I also added it in the environment variables PATH of windows 10.

the one in APPDATA 
and the one in C Program files

and it still can not import it

I am using visual studio code btw

**Comment by jongwook** [comment, marked as answer]
I think your visual studio code is configured to use a different python runtime from where your pip installs the package. In your terminal, `where python` and `where pip` will show the path of the Python and pip installation. (`Get-Command` instead of `where` if you happen to use Windows PowerShell) And then make sure [VSCode to use the same python interpreter](https://code.visualstudio.com/docs/python/environments#_working-with-python-interpreters).

**Comment by falves1** [comment]
I have the same issue using debian, I can transcribe files using bash, but not python. What am I missing

**Comment by GRL2025** [comment]
get the text from the video
